src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentValidator.java

```diff
@@ -68,7 +75,8 @@ public AttendanceAssignment validateCanvasAssignment(AttendanceAssignment assign
      */
     private boolean isAssignmentConfigurationSaved(AttendanceAssignment assignmentConfigurationFromSetup, AttendanceAssignment attendanceAssignmentSaved) {
 
-        return assignmentConfigurationFromSetup.getAssignmentPoints().equals(attendanceAssignmentSaved.getAssignmentPoints())
+        return assignmentConfigurationFromSetup.getAssignmentName().equals(attendanceAssignmentSaved.getAssignmentName())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1067)